compiler/resolution/preFold.cpp

```diff
@@ -537,33 +537,56 @@ static Expr* preFoldPrimInitVarForManagerResource(CallExpr* call) {
 
         cc->getCalls(refCall, valueCall, constRefCall);
 
-        isAnyRefOverloadPresent = (constRefCall || refCall);
```

Comment:
Good point, thanks

Reply:
Ah no, I think we need to keep these in the upper scope because the code following assumes that they can be set in either branch (even if they're only set in one).